### PR TITLE
Specify node version Netlify would use to deploy

### PR DESCRIPTION
**Pull request by UtkarshKr007**
Once this is merged in, we can delete the node version from Netlify system environments.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+12.0.0
```
Comment (floogulinc) [inline .nvmrc]: We don't want to force it to use exactly version `12.0.0`. Setting this to simply `12` would have it use any version 12.

```suggestion
12
```

We may want to just use `lts/*` which has it use the latest LTS version.
```suggestion
lts/*
```

**Review by floogulinc — CHANGES_REQUESTED** [review]
Why did you pick `.nvmrc` rather than `.node-version`?

**Comment by UtkarshKr007** [comment]
[Netlify Docs](https://docs.netlify.com/configure-builds/manage-dependencies/#node-js-and-javascript) stated they use nvm to fetch the node version we use. Netlify says either `.node-version` or `.nvrmc` is fine, but from what I looked into it didn't seem nvm people added support for `.node-version` yet. These are the issues/PRs I found on nvm's repo regarding this:

https://github.com/nvm-sh/nvm/issues/794
https://github.com/nvm-sh/nvm/pull/1625
https://github.com/nvm-sh/nvm/issues/2292

**Comment by floogulinc** [comment]
From @UtkarshKr007 

<blockquote>

[Netlify Docs](https://docs.netlify.com/configure-builds/manage-dependencies/#node-js-and-javascript) stated they use nvm to fetch the node version we use and either `.node-version` or `.nvrmc` is fine, but from what I looked into it didn't seem nvm has support for `.node-version` yet. These are the issues/PRs I found on nvm's repo regarding this:

```
https://github.com/nvm-sh/nvm/issues/794
https://github.com/nvm-sh/nvm/pull/1625
https://github.com/nvm-sh/nvm/issues/2292
```

</blockquote>

**Review by floogulinc — APPROVED** [review]

**Pull request merged**